amp/system-oracle/README.md

```diff
@@ -0,0 +1,46 @@
+# System image build with Oracle Database support
+
+
+## Prerequisites
+
+In order to build the image with Oracle Database support,
+you will need to download and add the binaries inside the directory `oracle-client-files`.
+
+Download the bellow instant clients files from the version **12.2** of oracle.
+
+You **MUST** download the client (basic-lite or basic), the odbc driver and the sdk:
+
+Download the following packages from [Oracle Technology Network](http://www.oracle.com/technetwork/database/features/instant-client/index-097480.html).
+
+* Instant Client Package - Basic or Basic Lite
+* Instant Client Package - SDK
+* Instant Client Package - ODBC driver
+
+Example:
+
+    * instantclient-basiclite-linux.x64-12.2.0.1.0.zip or instantclient-basic-linux.x64-12.2.0.1.0.zip
+    * instantclient-sdk-linux.x64-12.2.0.1.0.zip
+    * instantclient-odbc-linux.x64-12.2.0.1.0-2.zip
+
+## Instructions
+
+* Get the productized amp.yml template
+* Install the binary files as instructed above
+* Run
```

Comment:
"From the terminal, while in the "oracle-client-files" directory, run the following commands:

Reply:
Not really in the local oracle-client-files but OK I get it